.github/CODEOWNERS

```diff
@@ -457,3 +457,6 @@ Makefile                                              @thiagokokada
 /modules/programs/ion.nix                             @jo1gi
 
 /modules/services/plex-mpv-shim.nix                   @starcraft66
+
+/modules/programs/pistol.nix                          @mtoohey31
+/tests/modules/programs/pistol                        @mtoohey31
```

Comment:
These are supposed to be alphabetized (looks like a few slipped past us in the past). Please sort theses in the proper place in the file.

Reply:
Ah, mb I missed that, fixed in 8f4ebfb21d6012647d9cb78e15e071ffc14d96f5.